web/beacon-app/cypress/e2e/TopicDetail.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Topic Detail Page
+
+I want to navigate the Topic Detail Page
+
+Scenario: Navigating the Topic Detail Page
+
+Given I'm logged into Beacon
+And I navigate to the Topic Detail Page
+Then I should see the topic name in the header component
+#When I hover over the topic detail tooltip
```

Comment:
Cypress doesn't have a native way to test hover actions. This will be tested once I find a solution that works consistently.